src/components/ContentWrapperScroll.tsx

```diff
@@ -57,21 +57,32 @@ export const ContentWrapperScroll: FC<{
 
       <View
         style={{
+          flex: 1,
           backgroundColor: background,
+          marginTop: props.marginTop ?? 86,
+
           borderTopLeftRadius: 30,
           borderTopRightRadius: 30,
-          marginTop: props.marginTop ?? 86,
-          zIndex: 2,
-          flex: 1,
+
           shadowColor: "#000",
+          shadowOffset: {
+            width: 0,
+            height: 0,
+          },
           shadowOpacity: 0.2,
           shadowRadius: 8.3,
           elevation: 13,
-          overflow: "hidden",
         }}
       >
-        <ScrollView showsVerticalScrollIndicator={false}>
-          <View>{props.children}</View>
+        <ScrollView
+          style={{
+            borderTopLeftRadius: 30,
+            borderTopRightRadius: 30,
+            overflow: "hidden",
+          }}
+          showsVerticalScrollIndicator={false}
+        >
+          <View style={{ paddingBottom: 50 }}>{props.children}</View>
         </ScrollView>
```

Comment:
apparentemente il problema è causato da `overflow : "hidden"` dentro allo style della ScrollView. Rimettendolo dove era prima (nella view che wrappa la ScrollView) torna tutto a funzionare per qualche motivo a me ignoto.

Reply:
quell'overflow hidden è stato spostato lì sotto per un motivo specifico, dare la proprietà `overflow: "hidden"` a un componente con un'ombra su iOS taglia anche l'ombra, quindi bisogna avere una View con l'ombra fuori e una View con l'overflow hidden dentro così il contenuto non va fuori dagli angoli arrotondati, stupidissimo ma oh, è così.

a quanto pare mettere [overflow hidden sulle scrollview](https://stackoverflow.com/questions/41912313/element-overflow-hidden-in-react-native-android) rompe le cose su android? the more you know 🤷‍♂️ 
ho wrappato la scrollview in un'altra view ancora e ho messo lo stile a quella, un po' ridondante ma sembra funzionare

Reply:
Ah okay. Confermo che cosi funziona anche su android.